interfaces/apparmor/spec.go

```diff
@@ -57,6 +57,14 @@ type Specification struct {
 	//   }
 	suppressPtraceTrace bool
 	usesPtraceTrace     bool
+
+	// The home interface typically should have 'ix' as part of its rules,
+	// but specifying certain change_profile rules with these rules cases
+	// a 'conflicting x modifiers' parser error. Allow interfaces that
+	// require this type of change_profile rule to suppress 'ix' so that
+	// the calling interface can be used with the home interface. Ideally,
+	// we would not need this, but we currently do (LP: #1797786)
+	suppressHomeIx bool
```

Comment:
+1 on the change with the same suggestion to name this differently as in the PR that introduces ptrace methods and variables.

Reply:
Done